src/dev.js

```diff
@@ -1,11 +1,11 @@
 // @ts-check
 
+require("./lib/loadenv")
```

Comment:
Preferably this should be after we register the transpilers. However, since `loadenv` is vanilla JS there shouldn't be any issues as a result of putting it here.